js/dogs-javascript.js

```diff
@@ -0,0 +1,68 @@
+// alert info about the dog when user clicks on the image based on onclick in HTML
+function dogInfo(dog, breed, cost) {
+  const result = 'This dog is named ' + dog + '.  ' + dog + ' is a ' + breed + ' and will cost $' + cost + '.';
+  alert(result);
+}
+
+// calculate and alert the cummulative adoption costs and update the total cost in header when adopt button is clicked
+let totalCost = 0;
+function sumCost(cost) {
+totalCost = parseFloat((cost + totalCost).toFixed(2)); 
+alert('Total Adoption Fees: $' + totalCost.toFixed(2));
+
+//Update total cost in header with cummulative amount 
+
+$('.dog-button').click(function(){
+  $('.total').text('$'+totalCost.toFixed(2));
+});
+
+// set total cost in header to 0 when Clear Selection button is clicked
+$('.clear-dogs').click(function(){
+  $('.total').text('$0.00');
+});
+
+$('.clear-more-dogs').click(function(){
+  $('.total').text('$0.00');
+});
+
+}
+
+// change the border on dog tiles to Green and background to light blue when user mouses over the article
+$('article').on('mouseover', function (e) {
+  $(this).removeClass('noshow');
```

Comment:
When you addClass it might override what you have, so it might not be necessary to remove the old class. Kind a nice thing about jQuery